doc/lua_api.txt

```diff
@@ -2896,6 +2896,25 @@ For the following functions `x` can be either a vector or a number:
 * `vector.divide(v, x)`:
     * Returns a scaled vector or Schur quotient.
 
+For the following functions `a` is an angle in radians and `r` is a rotation
+vector ({x = <pitch>, y = <yaw>, z = <roll>}) where pitch, yaw and roll are
+in radians.
+
+* `vector.rotate(v, r):
```

Comment:
you're missing the ending ` on all of these

Reply:
and `<pitch>` might be parsed as HTML

Reply:
Adding the missing ` s right now.

Things that can be parsed as HTML are all over lua_api.txt, see e.g. [formspec docs](https://github.com/minetest/minetest/blob/454dbf83a9bf292910c1495a2aa49fd8b960c28f/doc/lua_api.txt#L438). I don't think me removing mine  individually would be of any value without a general 'no html syntax' rule for the docs.